### ISSUE TITLE
Redefine what an actual boat should be considered as

### ISSUE THREAD

**Issue by CounterCraftz** [GMDevinity/FloodIssues#556]
Players are able to progress to the fight phase with just 1 or multiple loose props. I don't think these should be considered as real boats because it encourages people to mow other players boats without any genuine effort. Maybe this has to deal with how the prop loose prop system works but it considered as a boat if the player has at least 2 props constrained. Spare props would be removed if the player built a boat and then decided to scatter single props around the map

**Comment by Lime1248** [comment]
I agree that loose props should not go through as real boats. I would think a general solution to those loose props that can float by themselves is change the buoyancy values (if that's possible) so they don't float or just remove them entirely. I think it also boils down to how much money players can earn when damaging props versus survival time. I can destroy a boat while sitting on 1 life preserver boat and make $1K with little to no effort which discourages the player who got destroyed to play the game any further. My solution would be to change how much a player earns through damage and how much a player earns through staying alive. While this is more of a massive change in how players earn money, it should go back to how the game is meant to be played which is survive to deal damage, not the other way around.
